### DISCUSSION TITLE
[Feature] Focus on text box in the add to album dialog

### DISCUSSION THREAD

**Discussion by vmoriaux** [immich-app/immich#9636] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Having a lot of albums, I always have to search for the album when adding pictures.
![image](https://github.com/immich-app/immich/assets/1343895/402f5a17-e905-4e90-8876-58d21a99d8ff)
I wish the "Add to album" dialog had the cursor/focus already set to the "Search" text box, so I can start typing immediately after opening the dialog.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
I believe this is now implemented
